### PR TITLE
Use constants to represent service names and values

### PR DESCRIPTION
**Pull request by Codebreake**

### CODE REVIEW

**Comment by Codebreake** [comment]
I deleted the other pull request and used the same branch as before to push the changes .

**Comment by Codebreake** [comment]
The third commit is the one containing the most recent changes.

File: src/components/sidepanel/SideBar.js [hunk 1]
```diff
@@ -109,27 +106,27 @@ export default class SideBar extends Component {
...
+                        [ services.ANDROID_USER_FLOWS_NAME, `/features/${services.ANDROID_USER_FLOWS_VALUE}`],
+                        [ services.ADMIN_USER_FLOWS_NAME, `/features/${services.ADMIN_USER_FLOWS_VALUE}`],
+                        [ services.SURVEYS_NAME , `/features/${services.SURVEYS_VALUE}`],
+                        [ services.RULES_NAME, `/features/${services.RULES_VALUE}`],
+                        [ services.LOANS_NAME, `/features/${services.LOANS_VALUE}`],
+                        [ services.USERS_NAME, `/features/${services.USERS_VALUE}`],
+                        [ services.AUTH_NAME, `/features/${services.AUTH_VALUE}`],
+                        [ services.RAILS_NAME, `/features/${services.RAILS_VALUE}`],
+                        [ services.COMMS_NAME, `/features/${services.COMMS_VALUE}`],
+                        [ services.COMMS_NAME, `/features/${services.APPROVAL_VALUE}`],
```
Comment (powermukisa) [inline src/components/sidepanel/SideBar.js]: this is repeated

File: src/components/sidepanel/SideBar.js [hunk 2]
```diff
@@ -144,25 +141,27 @@ export default class SideBar extends Component {
...
-                        [ "Assignment", '/completed/assignment'],
-                        [ "Dss", '/completed/dss'],
-                        [ "Kyc", '/completed/kyc'],
-                        [ "Attribution", '/completed/attribution'],
-                        [ "Settlement", '/completed/settlement'],
-                        [ "Verification", '/completed/verification'],
-                        [ "Lendingpartner", '/completed/lendingpartner'],
-                        [ "Provider Mediator (Legacy)", '/features/providermediator'],
-                        [ "Account (Legacy)", '/features/account']
+                        [ services.ANDROID_USER_FLOWS_NAME, `/completed/${services.ANDROID_USER_FLOWS_VALUE}`],
```
Comment (powermukisa) [inline src/components/sidepanel/SideBar.js]: this is not needed

File: src/components/sidepanel/SideBar.js [hunk 3]
```diff
@@ -144,25 +141,27 @@ export default class SideBar extends Component {
...
+                        [ services.ADMIN_USER_FLOWS_NAME, `/completed/${services.ADMIN_USER_FLOWS_VALUE}`],
```
Comment (powermukisa) [inline src/components/sidepanel/SideBar.js]: this is not needed

File: src/components/sidepanel/SideBar.js [hunk 4]
```diff
@@ -177,26 +176,27 @@ export default class SideBar extends Component {
...
-                          [ "Dss", '/development/dss'],
-                          [ "Kyc", '/development/kyc'],
-                          [ "Attribution", '/development/attribution'],
-                          [ "Settlement", '/development/settlement'],
-                          [ "Verification", '/development/verification'],
-                          [ "Lendingpartner", '/development/lendingpartner'],
-                          [ "Provider Mediator (Legacy)", '/features/providermediator'],
-                          [ "Account (Legacy)", '/features/account']
-
+                          [ services.ANDROID_USER_FLOWS_NAME, `/development/${services.ANDROID_USER_FLOWS_VALUE}`],
```
Comment (powermukisa) [inline src/components/sidepanel/SideBar.js]: this is not needed

File: src/components/sidepanel/SideBar.js [hunk 5]
```diff
@@ -302,4 +302,4 @@ export default class SideBar extends Component {
     )
   }
 
-}
+}
```
Comment (powermukisa) [inline src/components/sidepanel/SideBar.js]: new line

File: src/components/tests/createtestreport/CreateTestReport.js [hunk 6]
```diff
@@ -297,25 +299,27 @@ const CreateTestReport = (props) => {
...
-                <option value='assignment'>Assignment</option>
-                <option value='dss'>Dss</option>
-                <option value='kyc'>Kyc</option>
-                <option value='attribution'>Attribution</option>
-                <option value='settlement'>Settlement</option>
-                <option value='verification'>Verification</option>
-                <option value='lendingpartner'>Lending Partner</option>
-                <option value='providermediator'>Provider Mediator (Legacy)</option>
-                <option value='account'>Account (Legacy)</option>
+                      <option value={services.ANDROID_USER_FLOWS_VALUE}>{services.ANDROID_USER_FLOWS_NAME}</option>
```
Comment (powermukisa) [inline src/components/tests/createtestreport/CreateTestReport.js]: this is not needed

File: src/components/tests/createtestreport/CreateTestReport.js [hunk 7]
```diff
@@ -297,25 +299,27 @@ const CreateTestReport = (props) => {
...
+                <option value={services.ADMIN_USER_FLOWS_VALUE}>{services.ADMIN_USER_FLOWS_NAME}</option>
```
Comment (powermukisa) [inline src/components/tests/createtestreport/CreateTestReport.js]: not needed

File: src/components/tests/features/createfeature/CreateFeature.js [hunk 8]
```diff
@@ -13,6 +13,10 @@ import ServiceStatsDialog
   from "../../completedtests/servicestats/ServiceStatsDialog";
 import CreateFeatureDbHandler
   from "../featuresdbhandlers/CreateFeatureDbHandler";
+
+import * as services from "../../../../constants/Services"
+
+;
```
Comment (powermukisa) [inline src/components/tests/features/createfeature/CreateFeature.js]: move this

File: src/components/tests/createtestreport/CreateTestReport.js [hunk 9]
```diff
@@ -19,6 +19,8 @@ import {
 import TextField from "@material-ui/core/TextField/TextField";
 import {COMPLETED, NEW} from "../../../constants/ReportStatus";
 
+import * as services from "../../../constants/Services"
```
Comment (powermukisa) [inline src/components/tests/createtestreport/CreateTestReport.js]: add missing `;`

File: src/components/tests/features/createfeature/CreateFeature.js [hunk 10]
```diff
@@ -123,27 +127,27 @@ const CreateFeature = (props) => {
...
-                <option value='assignment'>Assignment</option>
-                <option value='dss'>Dss</option>
-                <option value='kyc'>Kyc</option>
-                <option value='attribution'>Attribution</option>
-                <option value='settlement'>Settlement</option>
-                <option value='verification'>Verification</option>
-                <option value='lendingpartner'>Lending Partner</option>
-                <option value='providermediator'>Provider Mediator (Legacy)</option>
-                <option value='account'>Account (Legacy)</option>
+                <option value={services.ANDROID_USER_FLOWS_VALUE}>{services.ANDROID_USER_FLOWS_NAME}</option>
```
Comment (powermukisa) [inline src/components/tests/features/createfeature/CreateFeature.js]: not needed

File: src/components/tests/features/createfeature/CreateFeature.js [hunk 11]
```diff
@@ -123,27 +127,27 @@ const CreateFeature = (props) => {
...
+                <option value={services.ADMIN_USER_FLOWS_VALUE}>{services.ADMIN_USER_FLOWS_NAME}</option>
```
Comment (powermukisa) [inline src/components/tests/features/createfeature/CreateFeature.js]: not needed

File: src/components/tests/features/createfeature/CreateFeature.js [hunk 12]
```diff
@@ -212,4 +216,4 @@ const mapDispatchToProps = dispatch => {
   };
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(CreateFeature);
+export default connect(mapStateToProps, mapDispatchToProps)(CreateFeature);
```
Comment (powermukisa) [inline src/components/tests/features/createfeature/CreateFeature.js]: new line

File: src/components/tests/updatetest/UpdateTestReport.js [hunk 13]
```diff
@@ -16,6 +16,7 @@ import {
 } from "../../../store/actions/authActions";
 import TextField from "@material-ui/core/TextField/TextField";
 import {blue} from "@material-ui/core/colors";
+import * as services from "../../../constants/Services"
```
Comment (powermukisa) [inline src/components/tests/updatetest/UpdateTestReport.js]: missing `;`

File: src/components/tests/updatetest/UpdateTestReport.js [hunk 14]
```diff
@@ -267,27 +268,29 @@ const UpdateTestReport = (props) => {
...
-                  <option value='kyc'>Kyc</option>
-                  <option value='attribution'>Attribution</option>
-                  <option value='settlement'>Settlement</option>
-                  <option value='verification'>Verification</option>
-                  <option value='lendingpartner'>Lendingpartner</option>
-                  <option value='providermediator'>Provider Mediator (Legacy)</option>
-                  <option value='account'>Account (Legacy)</option>
-                </select>
-              </div>
+                <option value={services.ANDROID_USER_FLOWS_VALUE}>{services.ANDROID_USER_FLOWS_NAME}</option>
```
Comment (powermukisa) [inline src/components/tests/updatetest/UpdateTestReport.js]: not needed

File: src/components/tests/updatetest/UpdateTestReport.js [hunk 15]
```diff
@@ -267,27 +268,29 @@ const UpdateTestReport = (props) => {
...
+                <option value={services.ADMIN_USER_FLOWS_VALUE}>{services.ADMIN_USER_FLOWS_NAME}</option>
```
Comment (powermukisa) [inline src/components/tests/updatetest/UpdateTestReport.js]: not needed

File: src/constants/Services.js [hunk 16]
```diff
@@ -50,7 +50,7 @@ export const SETTLEMENT_NAME = "Settlement";
 export const SETTLEMENT_VALUE = "settlement";
 
 export const VERIFICATION_NAME = "Verification";
-export const VERIFICATIONS_VALUE = "verification";
+export const VERIFICATION_VALUE = "verification";
```
Comment (powermukisa) [inline src/constants/Services.js]: good catch!

File: src/components/tests/updatetest/UpdateTestReport.js (hunk 13 above)
Comment (Codebreake) [reply src/components/tests/updatetest/UpdateTestReport.js]: DONE

File: src/components/tests/updatetest/UpdateTestReport.js (hunk 14 above)
Comment (Codebreake) [reply src/components/tests/updatetest/UpdateTestReport.js]: removed

File: src/components/tests/updatetest/UpdateTestReport.js (hunk 15 above)
Comment (Codebreake) [reply src/components/tests/updatetest/UpdateTestReport.js]: removed

File: src/components/sidepanel/SideBar.js (hunk 1 above)
Comment (Codebreake) [reply src/components/sidepanel/SideBar.js]: done

File: src/components/sidepanel/SideBar.js (hunk 2 above)
Comment (Codebreake) [reply src/components/sidepanel/SideBar.js]: removed

File: src/components/sidepanel/SideBar.js (hunk 3 above)
Comment (Codebreake) [reply src/components/sidepanel/SideBar.js]: removed

File: src/components/sidepanel/SideBar.js (hunk 5 above)
Comment (Codebreake) [reply src/components/sidepanel/SideBar.js]: done

File: src/components/tests/features/createfeature/CreateFeature.js (hunk 12 above)
Comment (Codebreake) [reply src/components/tests/features/createfeature/CreateFeature.js]: done

File: src/components/tests/createtestreport/CreateTestReport.js (hunk 7 above)
Comment (Codebreake) [reply src/components/tests/createtestreport/CreateTestReport.js]: removed

File: src/components/tests/createtestreport/CreateTestReport.js (hunk 6 above)
Comment (Codebreake) [reply src/components/tests/createtestreport/CreateTestReport.js]: removed

File: src/components/tests/features/createfeature/CreateFeature.js (hunk 8 above)
Comment (Codebreake) [reply src/components/tests/features/createfeature/CreateFeature.js]: done

File: src/components/tests/createtestreport/CreateTestReport.js (hunk 9 above)
Comment (Codebreake) [reply src/components/tests/createtestreport/CreateTestReport.js]: done

File: src/components/tests/features/createfeature/CreateFeature.js (hunk 10 above)
Comment (Codebreake) [reply src/components/tests/features/createfeature/CreateFeature.js]: removed

File: src/components/tests/features/createfeature/CreateFeature.js (hunk 11 above)
Comment (Codebreake) [reply src/components/tests/features/createfeature/CreateFeature.js]: removed

File: src/components/sidepanel/SideBar.js (hunk 4 above)
Comment (Codebreake) [reply src/components/sidepanel/SideBar.js]: removed

**Pull request merged**